src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncIteratorTests.cs

```diff
@@ -1008,6 +1302,37 @@ await using (var enumerator2 = enumerable.GetAsyncEnumerator())
             var comp = CreateCompilationWithTasksExtensions(new[] { source, AsyncStreamsTypes }, options: TestOptions.DebugExe);
             comp.VerifyDiagnostics();
             CompileAndVerify(comp, expectedOutput: "1 2 Stream1:3 1 2 Stream2:3 4 2 4 2 Done");
+
+            var comp2 = CreateCompilationWithTasksExtensions("", references: new[] { comp.EmitToImageReference() });
```

Comment:
>CreateCompilationWithTasksExtensions("", [](start = 24, length = 40)

Consider adding `symbolValidator=...` to the `CompileAndVerify` call above instead.